src/singletons/Updates.cpp

```diff
@@ -232,6 +232,12 @@ void Updates::installUpdates()
 
 void Updates::checkForUpdates()
 {
+#if !defined(Q_OS_WIN) && !defined(Q_OS_LINUX) && !defined(Q_OS_MACOS)
```

Comment:
I would also put this check into a separate function or a #define since it is still duplicated in the GeneralPage

Reply:
@fourtf I'd move it to Version.{hpp,cpp} and add something like Version::instance().isSupportedOS() and dispatch based on that. Is that a better approach in your opinion?